src/main/java/frc/robot/RobotContainer.java

```diff
@@ -274,8 +282,17 @@ private void configureTestBindings(SN_XboxController controller) {
   }
 
   private void configureAutoSelector() {
-    autoChooser.setDefaultOption("Preload Only",
-        new PreloadOnly(subStateMachine, subClimber, subDrivetrain, subElevator, subIntake, subShooter, subTransfer));
+    double preloadDelay = SmartDashboard.getNumber("Preload Only Auto", 0);
```

Comment:
This get number needs to be moved into the command as a lambda so it will dynamically update. I think currently it will not update except on robot boot